tools/run-dev.py

```diff
@@ -147,7 +147,8 @@
 if not os.path.exists("node_modules/.bin/handlebars") and os.path.exists("node_modules/handlebars"):
     print("Handlebars binary missing due to rebase past .gitignore fixup; fixing...")
     subprocess.check_call(["rm", "-rf", "node_modules/handlebars"])
-    subprocess.check_call(["yarn", "install", "--non-interactive", "--prefer-offline"])
+    YARN_BIN = "/srv/zulip-npm-cache/yarn/bin/yarn"
```

Comment:
I ended up changing that call, but the node in the node modules setup are still hard coded. That section of run-dev.py should probably be removed. The issue was with npm in the first place.